Tasks/SonarQubePostTest/PRCA/ReportProcessor-Module.psm1

```diff
@@ -225,6 +227,18 @@ function AnnotateIssuesWithRelativePath
     return $issues
 }
 
+#
+# File level issues have a 4-part component. Assembly level issues have a 3-part component.
+#
+function IsFileLevelIssue
```

Comment:
Do we want to cross reference that user bug here? At least might worth mentioning that PRCA will crash when the number of components is not 4.